CDCgov/prime-reportstream
# Exclude non-backend related files changes from triggering build

This PR excludes changes in 4 operations directories from triggering a backend build (`app|readme-assets|slack-boltjs-app|vpn`)

Test Steps:
1. Make changes to any of the listed excluded directories and make sure a build is not triggered

## Changes
- Exclude changes in operations directories `app|readme-assets|slack-boltjs-app|vpn` from triggering a build
- Output changed files in summary output

## Checklist

## Linked Issues
- Fixes #13561
